client/store-management-links/quick-link-category/test/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
This should be necessary as it's included in [`tests/js/setup-react-testing-library`](https://github.com/woocommerce/woocommerce-admin/blob/1e0061d0df8c177795c58ab9603af95f4aef6610/tests/js/setup-react-testing-library.js#L5).

Reply:
I missed that, thanks!